bcipy/helpers/triggers.py

```diff
@@ -212,6 +212,8 @@ class TriggerHandler:
     writing triggers and loading triggers from a txt file.
     """
 
+    encoding = 'utf-8'
```

Comment:
We are getting warnings on all of our `open` statements throughout BciPy. Should we have a constant in `system_utils` for this?

Reply:
I'm running python 3.8.10. I get warnings for other things when I run the tests or suite, but none for open. Can you paste the warning here?

Reply:
I'm seeing these as pylint warnings:

   `Using open without explicitly specifying an encoding`

Reply:
Thanks for fixing these!